app/presenters/publishing_api/ministers_index_presenter.rb

```diff
@@ -0,0 +1,61 @@
+module PublishingApi
+  class MinistersIndexPresenter
+    attr_accessor :item
+    attr_accessor :update_type
+
+    def initialize(item: nil, update_type: nil)
+      self.item = nil
+      self.update_type = update_type || "major"
+    end
+
+    delegate :content_id, to: :item
+
+    def content
+      content = BaseItemPresenter.new(
+        item,
+        title: 'ministers_index',
+        update_type: update_type,
+      ).base_attributes
+
+      content.merge!(
+        base_path: base_path,
+        details: details,
+        document_type: 'ministers_index',
+        rendering_app: Whitehall::RenderingApp::WHITEHALL_FRONTEND,
+        schema_name: "ministers_index",
+      )
+
+      if I18n.locale.to_s == "cy"
+        content.merge!(
+          PayloadBuilder::Routes.for(base_path, additional_routes: ["cy"])
+        )
+      else
+        content.merge!(
+          PayloadBuilder::Routes.for(base_path)
+        )
+      end
```

Comment:
If we follow the suggestions in the `base_path` method, this code can be simplified quite a bit:

```suggestion
      content.merge!(PayloadBuilder::Routes.for(base_path))
```